Tests/Microsoft.AppCenter.Test.Functional/Analytics/TrackEventTest.cs

```diff
@@ -68,9 +72,9 @@ public async Task TrackEventWithPropertiesAsync()
 
             // Verify.
             Assert.Equal("POST", httpNetworkAdapter.Method);
-            var actualEventName = (string)httpNetworkAdapter.JsonContent["logs"][0]["name"];
+            var actualEventName = (string)httpNetworkAdapter.JsonContent.SelectTokens($"$.logs[?(@.type == 'event')]").ToList()[0]["name"];
```

Comment:
in iOS, the start session log is usually in the same batch as the event.
on Android timing of the test is different and they are in 2 separate batches.

Reply:
This should be mentioned in the code. It is useful.